crates/bevy_pbr/src/meshlet/from_mesh.rs

```diff
@@ -283,19 +273,14 @@ fn simplify_meshlet_groups(
         }
     }
 
-    // Allow more deformation for high LOD levels (1% at LOD 1, 10% at LOD 20+)
-    let t = (lod_level - 1) as f32 / 19.0;
-    let target_error_relative = 0.1 * t + 0.01 * (1.0 - t);
-    let target_error = target_error_relative * mesh_scale;
-
     // Simplify the group to ~50% triangle count
     // TODO: Simplify using vertex attributes
     let mut error = 0.0;
     let simplified_group_indices = simplify(
         &group_indices,
         vertices,
         group_indices.len() / 2,
-        target_error,
+        f32::MAX,
         SimplifyOptions::LockBorder | SimplifyOptions::Sparse | SimplifyOptions::ErrorAbsolute,
```

Comment:
> Context: https://github.com/bevyengine/bevy/discussions/14998#discussioncomment-10508903 , section "What is a border?". 

I think that meshoptimizer's [definition of border vertex](https://github.com/zeux/meshoptimizer/blob/89ab317a96734460c0482c4ca41537611692e7e2/src/simplifier.cpp#L282) can be relaxed for Nanite. You could provide locked vertices (`const unsigned char* vertex_lock`) to e.g. [meshopt_simplifyWithAttributes()](https://github.com/zeux/meshoptimizer/blob/89ab317a96734460c0482c4ca41537611692e7e2/src/simplifier.cpp#L1781). This might lead to visual artefacts, but wouldn't that be an issue with an error metric instead?

Reply:
I'm not sure what open edge means in this context. But check the comment I left in that thread - I'm not sure that we're on the same page about which vertices need locking.